robot/impl/data/cfgtestuntrustedenv.json

```diff
@@ -0,0 +1,10 @@
+{
+    "modules": [
+        {
+            "name": "c1",
+            "type": "camera",
+            "model": "fake",
+            "attributes": {}
+        }
```

Comment:
This is not a module config, so it would fail in testing regardless of any new code you're adding. You may want to look at module/testmodule and just directly use the config (and module itself) in there.